staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation.go

```diff
@@ -82,7 +84,7 @@ func Compile(s *schema.Structural, isResourceRoot bool) ([]CompilationResult, er
 	for i, rule := range celRules {
 		var compilationResult CompilationResult
 		if len(strings.TrimSpace(rule.Rule)) == 0 {
-			compilationResult.Error = &Error{ErrorTypeRequired, "rule is not specified"}
+			continue
```

Comment:
now that we have the possibility of Compile returning an item that sets neither Error nor Program, should probably `continue` if we encounter a nil Program in `Validator#validateExpressions` as well

Reply:
Good point. I've cleaned this up and updated the godoc on Compile to explain.